Makefile

```diff
@@ -13,3 +13,7 @@ gen-doc:
 
 gen-docs:
 	docker run -v $(shell pwd):/go/src/app sigdocs
+
+test-unit:
+	docker build -t sigdocs-test -f generator/Dockerfile.test generator
+	docker run sigdocs-test
```

Comment:
We can likely rm the running container after completion. No need to leave cruft around.